challenge_6/python/mjuiuc/test.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+import time
+from Solution import solution
+
+
+
+class OutcomesTest(unittest.TestCase):
+
+	def test1(self):
+		tlist = [1,2,3,4,5,8,9,10]
+		case1 = solution()
+		t1 = time.time()
+		out = case1.compRange(tlist)
+		t2 = time.time()
+		print "Runtime for case1: " + str(t2-t1) + " seconds"
```

Comment:
I believe the pep 8 styleguide implies that spaces should go around all binary operators like " - " and " % "